src/controllers/loginController.js

```diff
@@ -0,0 +1,29 @@
+const loginService = require('../services/registration');
+
+const loginController = (req, res) => {
+  if (req.body.username && req.body.password) {
+    loginService.authorizeUser(JSON.stringify(req.body))
+      .then((data) => {
+        if (data === false) {
+          const errorMessage = {
+            message: 'Incorrect Username and/or Password!',
+          };
+          res.status(404).send(errorMessage);
+        } else {
+          const dataJSON = {
+            userId: data,
+          };
+          res.status(200).json(dataJSON);
+        }
+      });
+  } else {
+    const errorMessage = {
+      message: 'Please enter a Username and a Password!',
```

Comment:
This error message is not the same as above, this gives room for behavioural attacks. Unify the messages.